lib/xlat_tables_v2/aarch64/xlat_tables_arch.c

```diff
@@ -201,12 +201,12 @@ void init_xlat_tables_arch(unsigned long long max_pa)
 			/* Inner & outer non-cacheable non-shareable. */\
 			tcr = TCR_SH_NON_SHAREABLE |			\
 				TCR_RGN_OUTER_NC | TCR_RGN_INNER_NC |	\
-				(64 - __builtin_ctzl(PLAT_VIRT_ADDR_SPACE_SIZE));\
+				(64 - __builtin_ctzll(PLAT_VIRT_ADDR_SPACE_SIZE));\
```

Comment:
This change has to be made in lib/xlat_tables_v2/aarch32/xlat_tables_arch.c as well.